.github/workflows/nextjs.yml

```
@@ -1,93 +1,96 @@
-  deploy:
-    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v5
+name: Deploy Next.js site to Pages
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)